invenio/modules/news/encoder.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
```

Comment:
Use standard encryption for database columns from SQLAlchemy-Utils (see e.g. oauthserver.models). If you need to generate tokens then have a look to http://pythonhosted.org/itsdangerous/#url-safe-serialization